index.html

```diff
@@ -242,7 +242,7 @@ <h3><span><img src="images/Rectangle 57.png" alt=""></span>&nbsp; Skills</h3>
                 </span>
                 <input  type="email" placeholder="Email address">
                 <textarea name="comments" id="" cols="30" rows="3" placeholder="Write me something..."></textarea>
```

Comment:
- [ ] You have done great so far. Your design looks good but there are some project requirements you missed. :+1: .

> Collect data by using the [Formspree service](https://formspree.io/html).
- - According to the project requirement, you are supposed to have a post method on your form with an action that collects the form data and sends to Formspree, :+1: I'll suggest you follow the Formspree link and create an account, then get your lint and add to your from action. :+1: :+1: 

>A user cannot submit the form if one or more of the fields is empty.
- - According to to the requirements, all the input fields are required which means all the inputs need to have the `required` attribute, I'll suggest you add this attribute. :+1: 

> Name field accepts up to 30 chars. Message field accepts up to 500 chars.
- - According to this requirement, the name field and the message input should have a maximum character attribute `maxlength`, I'll suggest you add these to the correct inputs. :+1: :smile: